src/screens/Dashboard/CompactTickets.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {useState} from 'react';
+import {StyleSheet} from '@atb/theme';
+import * as Sections from '@atb/components/sections';
+import {
+  ShortTicketInfoView,
+  mapToUserProfilesWithCount,
+} from '../Ticketing/Ticket/TicketInfo';
+import {isPreactivatedTicket, isInspectableTicket} from '@atb/tickets';
+import {getValidityStatus} from '@atb/screens/Ticketing/Ticket/utils';
+import useInterval from '@atb/utils/use-interval';
+import {
+  useHasEnabledMobileToken,
+  useMobileTokenContextState,
+} from '@atb/mobile-token/MobileTokenContext';
+import {findReferenceDataById} from '@atb/reference-data/utils';
+import {useFirestoreConfiguration} from '@atb/configuration/FirestoreConfigurationContext';
+import {
+  filterActiveOrCanBeUsedFareContracts,
+  isValidRightNowFareContract,
+  useTicketState,
+} from '@atb/tickets';
+import ThemeText from '@atb/components/text';
+import {TicketsTexts, useTranslation} from '@atb/translations';
+
+type Props = {
+  onPressDetails?: (orderId: string) => void;
+};
+
+const CompactTickets: React.FC<Props> = ({onPressDetails}) => {
+  const itemStyle = useStyles();
+
+  // TODO: Verify if this is the correct way to update time-to-time the expiration time of the tickets.
+  const [now, setNow] = useState<number>(Date.now());
+  useInterval(() => setNow(Date.now()), 1000);
+
+  const {fareContracts, customerProfile} = useTicketState();
+  const activeFareContracts = filterActiveOrCanBeUsedFareContracts(
+    fareContracts,
+  ).sort(function (a, b): number {
+    const isA = isValidRightNowFareContract(a);
+    const isB = isValidRightNowFareContract(b);
+
+    if (isA === isB) return 0;
+    if (isA) return -1;
+    return 1;
+  });
+
+  // TODO: Double check if this is how it should be implemented, using the customer profile!
+  const hasActiveTravelCard = !!customerProfile?.travelcard;
+  const hasEnabledMobileToken = useHasEnabledMobileToken();
+  const {
+    deviceIsInspectable,
+    isError: mobileTokenError,
+    fallbackEnabled,
+  } = useMobileTokenContextState();
+  const {tariffZones, userProfiles, preassignedFareproducts} =
+    useFirestoreConfiguration();
+  const {t, language} = useTranslation();
+
+  return (
+    <>
+      <ThemeText
+        type="body__secondary"
+        color="secondary"
+        style={itemStyle.sectionText}
+        accessibilityLabel={t(TicketsTexts.header.title)}
+      >
+        {t(TicketsTexts.header.title)}
+      </ThemeText>
+      {activeFareContracts?.map((fareContract, index) => {
+        const travelRights =
```

Comment:
I think it's totally fine to extract this as its own component in the same file. It's easier to separate responsibilities and change over time. Here we essentially have 2 setup/initializer phases. One in the "List" component and one in the "Item" component. Further down we use a lot of variables, it's harder to control "which is closures" and which is dependencies to the sub component.